cw-log-config.txt

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
What's this file used for?

Reply:
I use this file in the boot script (maybe should find another home for it):
```
# Set-up log records on Cloudwatch:
curl https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py -O
python3 ./awslogs-agent-setup.py --region us-east-1 < ./cw-log-config.txt
```
to set the cloudwatch logs agent up, allows me to send the emails.